File-Formats/PDF/Convert-HTML-To-PDF/azure.md

```diff
@@ -174,139 +162,134 @@ A complete work sample for converting an HTML to PDF in Azure App service on Lin
      settings.CommandLineArguments.Add("--no-sandbox");
      settings.CommandLineArguments.Add("--disable-setuid-sandbox");
-     
      //Assign Blink settings to the HTML converter.
      htmlConverter.ConverterSettings = settings;
- 
      //Convert URL to PDF.
      PdfDocument document = htmlConverter.Convert("https://www.syncfusion.com");
- 
      MemoryStream stream = new MemoryStream();
- 
      //Save and close a PDF document. 
      document.Save(stream);
- 
      return File(stream.ToArray(), System.Net.Mime.MediaTypeNames.Application.Pdf, "URL_to_PDF.pdf");
 }
 
-{% endhighlight %}
+   {% endhighlight %}
 
-Build and run the sample in the docker, it will pull the Linux docker image from the docker hub and run the project. Now, the webpage will open in the browser. Click Export to PDF option to convert the Syncfusion webpage to a PDF.
+7. Build and run the sample in the docker, it will pull the Linux docker image from the docker hub and run the project. Now, the webpage will open in the browser. Click Export to PDF option to convert the Syncfusion webpage to a PDF.
 ![Convert HTMLToPDF Azure Docker Step6](htmlconversion_images/AzureDocker2.png)
 
-By executing the program, you will get the PDF document as follows.
+8. By executing the program, you will get the PDF document as follows.
 ![Convert HTMLToPDF Azure Docker Step7](htmlconversion_images/htmltopdfoutput.png)
 
-### Deploy the container to Azure container instance:
+## Deploy the container to Azure container instance:
 
-Create a publish target to deploy the docker image to Azure. 
+1. Create a publish target to deploy the docker image to Azure. 
 ![Convert HTMLToPDF Azure Docker Step8](htmlconversion_images/AzureDocker3.png)
 
-Create Azure App Service with resource group, hosting plan, and container registry. 
+2. Create Azure App Service with resource group, hosting plan, and container registry. 
 ![Convert HTMLToPDF Azure Docker Step9](htmlconversion_images/AzureDocker4.png)
 
-Publish the docker image to Azure container instance.
+3. Publish the docker image to Azure container instance.
 ![Convert HTMLToPDF Azure Docker Step9](htmlconversion_images/AzureDocker5.png)
 
-It will push the docker image to the Azure container registry and deploy it to the Azure container instance.
+4. It will push the docker image to the Azure container registry and deploy it to the Azure container instance.
 ![Convert HTMLToPDF Azure Docker Step9](htmlconversion_images/AzureDocker6.png)
 
-After successful deployment, it will open the Azure website in the browser.
+5. After successful deployment, it will open the Azure website in the browser.
 ![Convert HTMLToPDF Azure Docker Step9](htmlconversion_images/AzureDocker6.png)
 
-Click Export to PDF option to convert the Google webpage to a PDF. You will get the PDF document as follows.
+6. Click Export to PDF option to convert the Google webpage to a PDF. You will get the PDF document as follows.
```

Comment:
Modify like this  - Click the button to convert Syncfusion webpage to a PDF document. You will get the PDF document as follows.

Reply:
Modified